medmodels/medrecord/_overview.py

```diff
@@ -52,14 +50,17 @@ def extract_attribute_summary(
         Dict[MedRecordAttribute, Union[TemporalAttributeInfo, NumericAttributeInfo,
             StringAttributeInfo]: Summary of node or edge attributes.
     """
-    data = pl.DataFrame(data=[{"id": k, **v} for k, v in attribute_dictionary.items()])
+    list_of_dicts = [{**v} for v in attribute_dictionary.values()]
 
-    data_dict = {}
+    data = {}
+    for dictionary in list_of_dicts:
```

Comment:
There should no need of assigning that variable, right? 
```suggestion
    data = {}
    for dictionary in attribute_dictionary.values():
```

Reply:
True!